### PR TITLE
GFORMS-2518: Add a delete link to remove a snapshot that is no longer…

### PR DESCRIPTION
**Pull request by lxol**
… required

### CODE REVIEW

File: app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  messages: Messages,
+  l:LangADT
+)
+
+@backLink = @{
+  new BackLink(href = backUrl, content = new Text(messages("linkText.back")))
+}
+
+@{
+    val customContent = Html(s"""
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html:37]: What is this, why don't you use html directly?

File: app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_acknowledgement.scala.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  messages: Messages,
+  l:LangADT
+)
+
+@backLink = @{
+  new BackLink(href = backUrl, content = new Text(messages("linkText.back")))
+}
+
+@{
+    val customContent = Html(s"""
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_acknowledgement.scala.html:36]: Same here? Why not to use html directly?

File: app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala [hunk 3]
```diff
@@ -852,4 +865,37 @@ class TestOnlyController(
...
+          snapshotId,
+          actionUrl,
+          backUrl
+        )
+      ).pure[Future]
+    }
+
+  def deleteSnapshot(snapshotId: SnapshotId) =
+    controllerComponents.actionBuilder.async { implicit request =>
+      import SnapshotForms._
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala:885]: Would it be possible to delete all duplications of this line and put a single import
```
import uk.gov.hmrc.gform.testonly.SnapshotForms._
```
at the top instead?

**Review by josef-vlach — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html (hunk 1 above)
Comment (lxol) [reply app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html:37]: but I need to supply it to govuk_wrapper as argument. I create a variable with html.
```    govuk_wrapper(
 ...
      content = customContent,
  ...
    )
```

Comment (josef-vlach) [reply app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html:37]: Just do
```
@customContent = {
     <style>...
}
```
And replace all `$` with `@` and you almost good to go.

Comment (lxol) [reply app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_delete_confirmation.scala.html]: ok, I updated a PR

File: app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_page.scala.html [hunk 4]
```diff
@@ -53,19 +52,19 @@
...
 
     <div class="button-container">
-    <form action="$actionUrl" method="post">
+    <form action="@actionUrl" method="post">
 
         <p class="govuk-body">Snapshot description:</p>
         <div id="description-hint" class="govuk-hint">
           <pre>
-$description.value
+@description.value
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/views/hardcoded/pages/snapshot_page.scala.html:60]: Can we indent this (and all others instances below) correctly?

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**